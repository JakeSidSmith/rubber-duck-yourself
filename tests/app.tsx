import App from '^/app';
import React from 'react';
import renderer from 'react-test-renderer';

describe('app.tsx', () => {
  it('matches snapshot', () => {
    const tree = renderer.create(<App />);

    expect(tree).toMatchSnapshot();
  });
});
