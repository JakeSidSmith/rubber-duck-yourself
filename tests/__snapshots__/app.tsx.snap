// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app.tsx matches snapshot 1`] = `
<p>
  Hello, World!
</p>
`;
